src/platform/bizhawk/bizinterface.c

```diff
@@ -54,9 +54,9 @@ typedef struct
 	color_t vbuff[GBA_VIDEO_HORIZONTAL_PIXELS * GBA_VIDEO_VERTICAL_PIXELS];
 	void* rom;
 	struct VFile* romvf;
-	uint8_t bios[16384];
+	uint8_t bios[0x4000];
 	struct VFile* biosvf;
-	uint8_t sram[131072 + 16];
+	uint8_t sram[0x20000 + 16];

```

Comment:
Why not

```suggestion
	uint8_t sram[0x20010];

```
?

Reply:
0x20000 is the SRAM size (in hex as it's a nice power of 2) and 16 is some arbitrary amount of bytes extra for RTC.